src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -1840,28 +1844,13 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
   }
 
   private exitPostDonationSuccess(donation: Donation, stripe_donation_method: string|undefined) {
-    var action: string;
-    switch (stripe_donation_method) {
-        case 'card':
-          action = `stripe_card_payment_success`;
-          break;
-        case 'prb': // todo - check what value is actually emitted from the stripe element in staging and adjust this line.
-          action = 'stripe_prb_payment_success';
-          break;
-        default:
-            if (this.selectedSavedMethod) {
-                action = 'stripe_card_payment_success';
-            }
-            action = 'unknown_method_'+ stripe_donation_method+'_payment_success';
-            break;
-    }
 
-    console.log(stripe_donation_method);
+    const stripeMethod = stripe_donation_method || 'undefined';
 
     this.matomoTracker.trackEvent(
       'donate',
-      action,
-      `Stripe Intent processing or done for donation ${donation.donationId} to campaign ${donation.projectId}`,
+      'stripe_card_payment_success',
```

Comment:
@bdsl I was thinking we would still vary this to `stripe_prb_payment_success` for easier reporting of headline stats (completion numbers on wallets), while also using the below addition to help "zoom in" on any reported issues.

Reply:
Right - done in https://github.com/thebiggive/donate-frontend/commit/86713a524bb3ec947dfaf9657a9c9f95db3bf6d5